app/main/view.ts

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+import { BrowserView, BrowserWindow, app } from 'electron';
+
+import ConfigUtil = require('../renderer/js/utils/config-util');
+const shouldSilentWebview = ConfigUtil.getConfigItem('silent');
+
+export interface ViewProps {
+	index: number;
+	url: string;
+	name: string;
+	nodeIntegration: boolean;
+	preload: boolean;
+}
+
+export class View extends BrowserView {
+	index: number;
```

Comment:
Should we make an interface for this also?

Reply:
Not required since we haven't used it anywhere.